highway_env/envs/common/observation.py

```diff
@@ -187,6 +187,8 @@ def observe(self) -> np.ndarray:
         # Reorder
         df = df[self.features]
         obs = df.values.copy()
+        if not self.absolute:
+            obs[0,1]=0
```

Comment:
As I mentioned in #66 , I am not comfortable with treating x and y differently.
You can still do this manually for now, after the call to `obs, _, _, _ = env.step(action)`, and I will try to add the longitudinal/lateral lane coordinates as possible features soon.

Reply:
I undertand your point of view. I'm ok with that